src/Common/tests/System/Diagnostics/RemoteExecutorTestBase.cs

```diff
@@ -139,6 +137,7 @@ private static RemoteInvokeHandle RemoteInvoke(MethodInfo method, string[] args,
             }
             else
             {
+                //Desktop activation
```

Comment:
If we need it to work on desktop then we will need to make some changes here, because right now this code path will never run, because `File.Exists(Process.GetCurrentProcess().MainModule.FileName)` will always return true.

We can leave it for now, but we'll just have to keep that in mind when we turn desktop tests back on.